src/printer.js

```diff
@@ -1074,7 +1074,7 @@ function genericPrintNoParens(path, options, print) {
               n.selfClosing ? line : softline
             ])
           ),
-          n.selfClosing ? "/>" : ">"
+          n.selfClosing ? "/>" : ">",
```

Comment:
I actually don't either yet. I started to a while back but it was too unstable, it's probably fine now. I'm fine with waiting a little longer until we enforce it (need to figure out tooling first anyway)